lstore/table.py

```diff
@@ -401,5 +399,12 @@ def write_record(self, record: Record, rid: int) -> bool:
         return True
 
 
-    def read_record(self) -> bool:
-        pass
+    def read_record(self, key):
+        record = Record(key, 0, "00000", [1, 2, 3, 4, 5, 6])
```

Comment:
Is this just testing the functionality for now?

Reply:
Yes, just using as a tester.